docs/administration/configuration/configuring-storage-volumes/index.md

```diff
@@ -0,0 +1,194 @@
+# Configuring storage volumes
+
+As described in the [Deploying at scale](../../deployment/deploying-at-scale/index.md) documentation page, REANA needs a shared filesystem to store the workflows' workspaces.
+It is also possible to configure a second storage volume to be used by REANA's infrastructure components, such as the database and the message broker.
+
+## Configuration
+
+Volumes can be configured with two Helm values:
+
+- [`shared_storage`](https://github.com/reanahub/reana/tree/master/helm/reana) holds the configuration for the shared filesystem needed by REANA.
+  It is mandatory to configure this volume.
+- [`infrastructure_storage`](https://github.com/reanahub/reana/tree/master/helm/reana) configures the second optional volume used by the database and the message broker.
+  If not configured, the shared storage volume is used instead.
+  It has the same settings as [`shared_storage`](https://github.com/reanahub/reana/tree/master/helm/reana).
```

Comment:
(Second thought: since we would like to promote best practices here, perhaps we could always copy the `shared_storage` example and the `infrastructure_storage` example inside the same snippet?

This will allow us to:

- advertise best practices for "scaling out";
- advertise the sizes that one is huge one is smaller;
- and, the section about migrating from one volume to two volumes will make more sense.  (As in, the above was recommended way, but if you have an older cluster, you can upgrade as follows... kind of thinking.)

Reply:
I don't like too much repeating two times the same example for both the shared and the infrastructure volume, not only because this makes the page longer, but also because I think it gives the false impression that both the shared and the infrastructure storage need to have the same backend (e.g. both of them as CephFS shares).

I have reformulated the last phrase a bit:
"In the configuration examples below we'll use the `shared_storage` volume, but the `infrastructure_storage` volume can be configured in exactly the same way. Note that typically, the shared storage volume can be huge (even tens of terabytes), while the shared storage volume can be as small as a few gigabytes, depending also on whether you are using an external database or not."

Reply:
Yes I agree that my suggestion would seem to say that using different backends for different shared volumes is not possible, which would be very misleading... OTOH, if we don't display a full "scalable setup", then people might simply not use it.  Perhaps we could solve it by having a new subsection with a "complete example" which could actually show the infra volume being on NFS and the runtime shared volume being on CephFS as one possible robust setup? (Could be done also later.)